### ISSUE TITLE
@GraphQLIgnore is not available on field with lombok

### ISSUE THREAD

**Issue by whalevii-xujifeng** [leangen/graphql-spqr#241]
@GraphQLIgnore is not available on field with lombok
```java
@GraphQLType
@Getter
@Setter
class A{
  @GraphQLIgnore 
  private Long id;
}
```
field id is still exists in the schema

**Comment by MateuszGajowski** [comment]
@whalevii-xujifeng 
For your input type you have to annotate field with `@Setter(onMethod_= {@GraphQLIgnore})`.
For your output/response type you have to annotate field with `@Getter(onMethod_= {@GraphQLIgnore})`

**Comment by kaqqao** [comment]
Also see #160 due yo be released in 0.9.10
